rules/game_process.tex

```diff
@@ -75,6 +81,14 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
+At the scheduled time, the referee will announce that the transition from the \texttt{setup} to \texttt{initial} will be executed 60 seconds later.
+The referee shall call ``Initial in 60 seconds''. 
+The GameController will execute this transition automatically once the time has passed. 
+If both teams have left the field, the head referee can announce the transition earlier by calling ``Initial''.
```

Comment:
This sounds odd. I guess you are referring to the human team members and that the robot team members are on the field. Rewording needed I think.